concept-anchor.md

```diff
@@ -21,19 +21,24 @@ These organizations might have an existing strategy, such as Agile, Lean, or Dev
+b) Fostering productivity
+While the corporation's strategy may involve colocating employees who collaborate with each other regularly, it may still be a global corporation. Robust CI/CD and documentation facilitates collaboration and productivity across time zones and geographical locations. It eliminates barriers between teams and the production environment.
+* *Quality Focus* this is a major point for IS. Because of the expected submission by a variety of teams and individuals, extra care is taken to make sure that the submitted code is screened to be production quality. Comprehensive documentation for submitters, test coverage, CI/CD, static code analysis and other techniques are often used to screen code from strangers in the open source world, these techniques have great benefits for internal teams.
 
 ##### [SCRUM](https://en.wikipedia.org/wiki/Scrum_%28software_development%29)
-Tshorten the example and make it more concrete for SCRUM, a popular agile methodology, one could cite the following scenario:
+To shorten the example and make it more concrete for SCRUM, a popular agile methodology, one could cite the following scenario:
 Team A depends on Team B's code. Team A has a pressing priority for delivery, Team B has their own priorities, none of which is Team A, what to do? Using InnerSource, Team A can put the required code changes in to their own backlog and submit it to Team B, essentially removing block making things more "Agile".
 
 ### DevOps
 There is no fixed definition of DevOps but much of it is focused around integrating Software Development and other dependencies such as Operations (auxiliary support such as servers, security etc.) and other business to reduce time required to deliver value. So where Agile focuses on the actual software development, DevOps is seen to incorporate all the other areas involving delivery.
 
 One major concept with DevOps is not to fear change, to be able to remove the risks so that experiments on the system can be comfortably performed.
 
-InnerSource helps in this process by its emphasis on a CI/CD workflow as well as a culture to automate much of the risk management and checks as possible. The fact that their are multiple contributors also helps because the more eyeballs there are, the more perspectives there are on how risks are to be managed, this forces the operations team to design for the lowest common denominator, thus improving overall quality and experience for all
+InnerSource helps in this process by its emphasis on a CI/CD workflow as well as a culture to automate much of the risk management and checks as possible. The more contributors there are, the more eyeballs there are. The more eye balls there are, the more diverse perspectives there are on managing risk. These diverse perspectives help the operations team design for the lowest (and broadest?) common denominator thus improving the overall quality and experience for all.
```

Comment:
eye balls or eyeballs? :)

Reply:
I would edit my own edit: "(and broadest?)"
This was meant to ask a question to you @mishari 
If you think that those words are indeed relevant, I'd remove the () and ? before merging it in.

Reply:
> eye balls or eyeballs? :)

dictionary says eyeball :)